.travis.yml

```diff
@@ -1,40 +1,40 @@
 sudo: required
-
 services:
-  - docker
-
+- docker
 env:
-  - DOCKER_COMPOSE_VERSION=1.11.2
-
+- DOCKER_COMPOSE_VERSION=1.11.2
 language: python
-
 python:
-  - "3.4"
-
+- '3.6'
 install:
-  - sudo apt-get update
-  - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-engine
-  - sudo rm /usr/local/bin/docker-compose
-  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
-  - chmod +x docker-compose
-  - sudo mv docker-compose /usr/local/bin
-  - sudo docker --version
-  - sudo docker-compose --version
-
+- sudo apt-get update
+- sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-engine
+- sudo rm /usr/local/bin/docker-compose
+- curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname
+  -s`-`uname -m` > docker-compose
```

Comment:
It is from the previous line, but the Travis CI YAML styler did this. It also does this:

```
key:
- value
```

instead of 

```
key:
  - value
```

which I find odd.

I'll edit that line to be one long one instead.